mem/mem.go

```diff
@@ -46,17 +46,30 @@ type VirtualMemoryStat struct {
 	// https://www.centos.org/docs/5/html/5.1/Deployment_Guide/s2-proc-meminfo.html
 	// https://www.kernel.org/doc/Documentation/filesystems/proc.txt
 	// https://www.kernel.org/doc/Documentation/vm/overcommit-accounting
-	Buffers      uint64 `json:"buffers"`
-	Cached       uint64 `json:"cached"`
-	Writeback    uint64 `json:"writeback"`
-	Dirty        uint64 `json:"dirty"`
-	WritebackTmp uint64 `json:"writebacktmp"`
-	Shared       uint64 `json:"shared"`
-	Slab         uint64 `json:"slab"`
-	PageTables   uint64 `json:"pagetables"`
-	SwapCached   uint64 `json:"swapcached"`
-	CommitLimit  uint64 `json:"commitlimit"`
-	CommittedAS  uint64 `json:"committedas"`
+	Buffers        uint64 `json:"buffers"`
+	Cached         uint64 `json:"cached"`
+	Writeback      uint64 `json:"writeback"`
+	Dirty          uint64 `json:"dirty"`
+	WritebackTmp   uint64 `json:"writebacktmp"`
+	Shared         uint64 `json:"shared"`
+	Slab           uint64 `json:"slab"`
+	PageTables     uint64 `json:"pagetables"`
+	SwapCached     uint64 `json:"swapcached"`
+	CommitLimit    uint64 `json:"commitlimit"`
+	CommittedAS    uint64 `json:"committedas"`
+	HighTotal      uint64 `json:"hightotal"`
+	HighFree       uint64 `json:"highfree"`
+	LowTotal       uint64 `json:"lowtotal"`
+	LowFree        uint64 `json:"lowfree"`
+	SwapTotal      uint64 `json:"swaptotal"`
+	SwapFree       uint64 `json:"swapfree"`
+	Mapped         uint64 `json:"mapped"`
+	VMallocTotal   uint64 `json:"vmalloctotal"`
+	VMallocUsed    uint64 `json:"vmallocused"`
+	VMallocChunk   uint64 `json:"vmallocchunk"`
+	HugePagesTotal uint64 `json:"hugePagestotal"`
+	HugePagesFree  uint64 `json:"hugePagesfree"`
```

Comment:
hugePagestotal and hugePagesfree should be in all lowercase

Reply:
good catch